controllers/reviews.controller.js

```diff
@@ -29,3 +30,14 @@ exports.getCommentsByReviewID = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  review_id = req.params.review_id;
+  newComment = req.body;
```

Comment:
VARIABLES WITHOUT DECLARATIONS?! straight to jail :/

Reply:
🚨🚔👮⚖️🧑‍⚖️⛓️😭